nuxeo-core/nuxeo-core-convert/src/main/java/org/nuxeo/ecm/core/convert/extension/Converter.java

```diff
@@ -50,4 +50,16 @@
      */
     BlobHolder convert(BlobHolder blobHolder, Map<String, Serializable> parameters) throws ConversionException;
 
+    /**
+     * @since 11.1
+     * To override with converter specific logic only if needed.
+     *
+     * {@inheritDoc Converter#convert(BlobHolder, Map)}
```

Comment:
`{@inheritDoc}` does not work like that, you cannot point to another method in the same type. Just repeat the doc instead.